server/game/player.js

```diff
@@ -159,7 +161,18 @@ class Player extends Spectator {
     }
 
     getNumberOfUsedPlots() {
-        return this.plotDiscard.length + this.usedPlotsModifier;
+        return this.plotDiscard.length + this.usedPlotsModifier + this.usedPlotsModifierByTrait.getValues().reduce((sum, entry) => sum + this.usedPlotsModifierByTrait.getCountForReference(entry), 0);
```

Comment:
Just to confirm this is intentional, this means "considered to have 1 additional <trait> plot in their used pile" counts toward the total number of plots in the used pile, not just when counting by that trait?

Reply:
Yes, having additional used plots (without a trait) added by Ricasso (for example) and having an additional CITY used plot added by Gulltown should be two additional used plots in total.